.travis.yml

```diff
@@ -25,12 +25,16 @@ branches:
     - /^amp-release-.*$/
     - /^revert-.*$/
 addons:
+  apt:
+    packages:
+      - protobuf-compiler
+      - python-protobuf
+  chrome: stable
```

Comment:
Interesting that `addons: chrome: stable` works. Travis docs says to use `addons: apt: sources: google-chrome-stable`  https://docs.travis-ci.com/user/reference/xenial/#third-party-apt-repositories-removed 

I wonder if that makes a difference at all?

Reply:
The main documentation at https://docs.travis-ci.com/user/chrome recommends `addons: chrome: stable`. I think `addons: apt` is for cases when other versions need to be manually installed.